lib/as_action_ocaml.ml

```diff
@@ -67,7 +67,7 @@ let compute_deps_ml = compute_deps `Ml
 
 let compile_mli
     ?(needs = []) ?(pkgs = nop) ?(args = []) ~ocamlc ~annot ~incs ~src () =
-  let ctx = As_ctx.v [`OCaml; `Compile; `Src `Mli] in
+  let ctx = As_ctx.v [`OCaml; `Compile; `Src `Mli; `Target `Byte] in
```

Comment:
Yes I think that's the right solution in fact we are already doing that kind of distinction for project [local paths](https://github.com/samoht/assemblage/blob/master/lib/as_part_unit.ml#L147). It may not be meaningful now but it may become in the future. End user that want to address the computation should use the context `` `[`OCaml; `Compile; `Src `Mli; ]`` (without the target specification) for now (this will work since contexts are matched by subsetting) as there is concretly no distinction between the two targets. The one that is selected is based purely on the availability of tools at hand.